src/services/refactors/moveToNewFile.ts

```diff
@@ -126,25 +139,39 @@ namespace ts.refactor {
 
         deleteUnusedOldImports(oldFile, toMove.all, changes, usage.unusedImportsFromOldFile, checker);
         deleteMovedStatements(oldFile, toMove.ranges, changes);
-
         updateImportsInOtherFiles(changes, program, oldFile, usage.movedSymbols, newModuleName);
 
         const imports = getNewFileImportsAndAddExportInOldFile(oldFile, usage.oldImportsNeededByNewFile, usage.newFileImportsFromOldFile, changes, checker, useEs6ModuleSyntax, quotePreference);
         const body = addExports(oldFile, toMove.all, usage.oldFileImportsFromNewFile, useEs6ModuleSyntax);
         if (imports.length && body.length) {
             return [
+                ...prologueDirectives,
                 ...imports,
                 SyntaxKind.NewLineTrivia as const,
                 ...body
             ];
         }
 
         return [
+            ...prologueDirectives,
             ...imports,
             ...body,
         ];
     }
 
+    function getPrologueDirectives(statements: readonly Statement[]): PrologueDirective[] {

```

Comment:
I think we might already have a utility for this.

Reply:
I didn't find a similar utility. There is a utility to find only "use strict" directive;